app/src/main/java/com/spendster/presentation/addExpenses/AddExpensesActivity.java

```diff
@@ -69,6 +82,32 @@ public void onClick(View view) {
         }
     }
 
+    private void addExpenses() throws ParseException {
```

Comment:
@SimonMaha this method should not throw. It should be in presenter and handled there, not on UI.